ocp_resources/datavolume.py

```diff
@@ -242,6 +242,14 @@ def wait(self, timeout=TIMEOUT_10MINUTES, failure_timeout=TIMEOUT_2MINUTES):
             status=PersistentVolumeClaim.Status.BOUND, timeout=timeout
         )
 
+    def is_deployed_by_populators(self):
```

Comment:
Set this function as `@property`
Rename to `deployed_by_populators`

Reply:
ack